packages/flutter/test/widgets/icon_data_test.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test('IconDataDiagnosticsProperty includes valueProperties in JSON', () {
+    IconDataProperty property = IconDataProperty('foo', const IconData(101010));
+    final Map<String, Object> valueProperties = property.toJsonMap(const DiagnosticsSerializationDelegate())['valueProperties'];
+    print(valueProperties);
```

Comment:
nooooooooooo

Reply:
Fixed in https://github.com/flutter/flutter/pull/35408. Sorry :(